kpops/cli/main.py

```diff
@@ -165,8 +165,7 @@ def is_in_steps(component: PipelineComponent) -> bool:
             else not is_in_steps(component)
         )
     ]
-    operation = "Including" if filter_type is FilterType.INCLUDE else "Excluding"
-    log.info(f"{operation} the following steps: {get_step_names(filtered_steps)}")
+    log.info(f"The following steps are executed:\n{get_step_names(filtered_steps)}")
```

Comment:
executed is a misleading term, e.g. when the user is running `kpops generate --template`

Reply:
Fair enough, changed it